extensions/parrot-translate/src/components.tsx

```diff
@@ -0,0 +1,145 @@
+import { Component } from "react"
+import { exec } from "child_process"
+import { COPY_TYPE, LANGUAGE_LIST } from "./consts"
+import { reformatCopyTextArray, truncate } from "./shared.func"
+import { IActionCopyListSection, IListItemActionPanelItem, IPreferences } from "./types"
+import { Action, ActionPanel, Clipboard, getPreferenceValues, Icon, Keyboard } from "@raycast/api"
+
+const preferences: IPreferences = getPreferenceValues()
+
+export function ActionCopyListSection(props: IActionCopyListSection) {
+    if (!props.copyText) {
+        return null
+    }
+
+    const SEPARATOR = "；"
+    const copyTextArray = props.copyText.split(SEPARATOR)
+    copyTextArray.length > 1 && copyTextArray.push(props.copyText)
+    const finalTextArray = reformatCopyTextArray(copyTextArray, 4)
+
+    const shortcutKeyEquivalent: Keyboard.KeyEquivalent[] = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"]
+
+    function changeTextCopyStyle(text: string): string {
+        const textArray: string[] = text.split(" ")
+        if (props.copyMode === COPY_TYPE.Uppercase) {
+            const SEPARATOR = "_"
+            return textArray
+                .map((text) => {
+                    return text.toUpperCase()
+                })
+                .join(SEPARATOR)
+        } else if (props.copyMode === COPY_TYPE.LowercaseCamelCase && textArray.length > 1) {
+            return textArray
+                .map((text, idx) => {
+                    if (idx === 0) return text.toLowerCase()
+
+                    const firstLetter = text.slice(0, 1).toUpperCase()
+                    return firstLetter + text.slice(1, text.length)
+                })
+                .join("")
+        }
+
+        return text
+    }
+
+    return (
+        <ActionPanel.Section>
+            {finalTextArray.map((textItem, key) => {
+                const title = changeTextCopyStyle(textItem.title)
```

Comment:
```suggestion
```
Redundant

Reply:
the title key content and value key content are not the same when the translated resulting length is too long. so I need to keep the title and value key.

Reply:
<img width="796" alt="image" src="https://user-images.githubusercontent.com/6928850/158006003-22cb5e43-a4ca-4b78-a799-809e1397b735.png">

The Title is: `Copy All`, The value key actually is `How are you；How Do You Do；Are You OK`

Reply:
Ah, fair!